### PR TITLE
Fix for error when deploying

### PR DESCRIPTION
**Pull request by ArielLima**
Ran into this error when deploying the project:

`The Deployment "aws-account-shredder" is invalid: spec.template.spec.containers[0].imagePullPolicy: Unsupported value: "always": supported values: "Always", "IfNotPresent", "Never"`

This PR simply updates the imagePullPolicy value from `always` to `Always`

### CODE REVIEW

File: deploy/deployment.yaml [hunk 1]
```diff
@@ -17,8 +17,8 @@ spec:
       serviceAccountName: aws-account-shredder
       containers:
         - name: aws-account-shredder
-          image: quay.io/agautam/aws-account-shredder:latest
-          imagePullPolicy: always
+          image: quay.io/app-sre/aws-account-shredder
```
Comment (fahlmant) [inline deploy/deployment.yaml:20]: why was latest removed?

File: go.mod [hunk 2]
```diff
@@ -35,5 +35,4 @@ require (
 	k8s.io/apimachinery v0.18.2
 	k8s.io/client-go v12.0.0+incompatible
 	sigs.k8s.io/controller-runtime v0.6.0
-	sigs.k8s.io/structured-merge-diff v0.0.0-20190525122527-15d366b2352e // indirect
```
Comment (fahlmant) [inline go.mod:38]: Why was this removed?

**Review by fahlmant — COMMENTED** [review]
Please also update the template to match the deployment image used

File: deploy/deployment.yaml (hunk 1 above)
Comment (ArielLima) [reply deploy/deployment.yaml:20]: I just copied what was on the [pipeline](https://gitlab.cee.redhat.com/service/app-interface/-/blob/master/data/services/osd-operators/cicd/saas/saas-aws-account-shredder.yaml#L30)

File: go.mod (hunk 2 above)
Comment (ArielLima) [reply go.mod:38]: This was removed automatically, I am unsure why that happened

Comment (fahlmant) [reply go.mod:38]: What did you run that removed this?

File: deploy/deployment.yaml (hunk 1 above)
Comment (fahlmant) [reply deploy/deployment.yaml:20]: The pipline just replaces the image name in our template. Our template uses latest as the tag. Please add latest back here

**Comment by rogbas** [comment]
@ArielLima PR check is failing, could you please :eyes: ?

**Review by fahlmant — APPROVED** [review]
/lgtm
